assets/js/base/context/event-emit/utils.ts

```diff
@@ -26,8 +26,16 @@ export enum responseTypes {
 }
 
 export enum noticeContexts {
-	PAYMENTS = 'wc/payment-area',
-	EXPRESS_PAYMENTS = 'wc/express-payment-area',
+	GLOBAL = 'wc/global',
+	CART = 'wc/cart',
+	CHECKOUT = 'wc/checkout',
+	PAYMENTS = 'wc/checkout/payments',
+	EXPRESS_PAYMENTS = 'wc/checkout/express-payments',
+	CONTACT_INFO = 'wc/checkout/contact-info',
```

Comment:
nitpick: should we use the full block name (minus block) for the context? so this would be 

```suggestion
	CONTACT_INFORMATION = 'wc/checkout/contact-information',
```